ObedZac/iREPORTER
# Ft delete specific redflag 162221049

# 162221049 delete specific red flag end point

### *What does this PR do*?

This PR creates the end point for the API to delete a specific red flag
the test for deleting a red flag should pass

### *Description of Task to be completed*?

The app should delete a redflag

### *How should this be manually tested*?

create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest n the test for deleting a red flag should pass
or run the app via python run.py / flask run and test on postman

### *What are the relevant pivotal tracker stories*?

162221049
